scripts/wrappers/common/utils.py

```diff
@@ -146,6 +146,37 @@ def get_dqlite_info():
     return info
 
 
+def get_etcd_info():
+    kube_apiserver_args = os.path.expandvars("${SNAP_DATA}/args/kube-apiserver")
+    with open(kube_apiserver_args, "r") as f:
+        kube_apiserver_args_content = f.read()
+        etcd_endpoints = []
+        for line in kube_apiserver_args_content.split("\n"):
+            if "etcd-servers" in line:
+                # list all etcd endpointss
+                if "=" in line:
+                    for endpoint in line.split("=")[1].split(","):
+                        if "//" in endpoint:
```

Comment:
As mentioned earlier, this is valid configuration but is not picked up from the current function
```
--etcd-servers http://10.0.0.1:2379
```